.eslintrc.js

```diff
@@ -0,0 +1,46 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true,
+    es6: true,
+  },
+  extends: ['plugin:react/recommended'],
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true,
+    },
+    ecmaVersion: 'latest',
+    sourceType: 'module',
+  },
+  plugins: ['prettier', 'react'],
+  rules: {
+    'import/no-extraneous-dependencies': 'off',
+    'import/prefer-default-export': 'off',
+    'lines-between-class-members': ['error', 'always'],
+    'no-comma-dangle': 'off',
+    'no-multiple-empty-lines': ['error', { max: 1 }],
+    'no-unneeded-ternary': 'error',
+    'react/jsx-sort-props': 'error',
+    'react/jsx-uses-react': 'error',
+    'react/jsx-uses-vars': 'error',
+    'react/function-component-definition': 'off',
+    'react/jsx-one-expression-per-line': 'off',
+    'react/jsx-filename-extension': [
+      1,
+      {
+        extensions: ['.js', '.jsx', '.ts', '.tsx'],
+      },
+    ],
+    'react/no-unknown-property': 'off',
+    'react/react-in-jsx-scope': 'off',
```

Comment:
<img width="1024" alt="image" src="https://user-images.githubusercontent.com/109110152/178598860-4e8528dd-c3a9-41a3-858a-15095d91a217.png">

Maybe we should do the second statement instead of turning it off explicitly?

Reply:
I can remove or change this.